submarine-server/server-core/src/main/java/org/apache/submarine/server/experiment/ExperimentManager.java

```diff
@@ -108,8 +108,10 @@ public Experiment createExperiment(ExperimentSpec spec) throws SubmarineRuntimeE
     spec.getMeta().getEnvVars().put(RestConstants.SUBMARINE_TRACKING_URI, url);
     spec.getMeta().getEnvVars().put(RestConstants.LOG_DIR_KEY, RestConstants.LOG_DIR_VALUE);
 
-    String lowerName = spec.getMeta().getName().toLowerCase();
+    String lowerName = spec.getMeta().getName().toLowerCase(); 
     spec.getMeta().setName(lowerName);
+    spec.getMeta().setExpID(id.toString().replaceAll("_", "-"));
```

Comment:
What's the reason of considering '-' rather than '_'? 

Supported character set of K8s: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set

Reply:
#630 describes that why I replace "_" with "-".

Reply:
Yes. But the label value could contain dashes (-), underscores (_), dots (.), and alphanumerics between. 

I think we should keep it, so that we can't convert the concept when use label selector. Any thoughts?

Reply:
+1 After discussed offline. Thanks @kevin85421 for the future design, I expected the refactor of experiment feature.